tests/gis_tests/geoapp/test_functions.py

```diff
@@ -148,12 +148,13 @@ def test_assvg(self):
     @skipUnlessDBFeature("has_BoundingCircle_function")
     def test_bounding_circle(self):
         qs = Country.objects.annotate(circle=functions.BoundingCircle('mpoly')).order_by('name')
-        self.assertAlmostEqual(qs[0].circle.area, 168.89, 2)
-        self.assertAlmostEqual(qs[1].circle.area, 135.95, 2)
+        self.assertAlmostEqual(qs[0].circle.area, 169, 0)
```

Comment:
@claudep, are these relaxations okay or would you do something different? PostGIS 2.3 is returning some different values. I can try to find the relevant change if you feel it's needed (just in case it might reveal a PostGIS bug).